### ISSUE TITLE
Crash when using Simple BC Pipes.

### ISSUE THREAD

**Issue by Dakedres** [Chainmail-Studios/Astromine#114]
I was messing around with [Simple BC Pipes,](https://www.curseforge.com/minecraft/mc-mods/simplepipes) I don't think I even had an astromine block placed down, when I got this crash 

log: https://paste.ee/p/YYgDM

I'm assuming it's some sort of compatability issue because of this line:
```
java.lang.NullPointerException: Unexpected error
	at com.github.chainmailstudios.astromine.common.lba.LBAFluidCompat.append(LBAFluidCompat.java:49)
```
Thanks!

**Comment by vini2003** [comment]
@shedaniel well I guess this clears our field test of LBA compat.

**Comment by Devan-Kerman** [comment]
yotefuckinhaw

**Comment by vini2003** [comment]
I have made it not crash anymore, in theory, however it is still having significant trouble working.

**Comment by shedaniel** [comment]
it was a good bet.

**Comment by vini2003** [comment]
Compatibility fixed as of #122.

**Comment by Dakedres** [comment]
Nice.
